server/config.json

```diff
@@ -51,27 +52,29 @@
             "column": "token_id"
           }
         ]
-      }
+      },
+      "notification": { "columns": ["log_addr"] }
     },
     {
       "enabled": true,
       "name": "bodies_transfer",
-      "sources": [{ "name": "sepolia", "start": "5716604" }],
+      "sources": [{ "name": "base_sepolia", "start": "10923234" }],
       "table": {
         "name": "bodies_transfer",
         "columns": [
           { "name": "from", "type": "bytea", "indexed": true },
           { "name": "to", "type": "bytea", "indexed": true },
           { "name": "token_id", "type": "numeric", "indexed": true },
           { "name": "log_addr", "type": "bytea", "indexed": true }
-        ]
+        ],
+        "index": [["block_num DESC", "tx_idx DESC", "log_idx DESC"]]
       },
       "block": [
         {
           "name": "log_addr",
           "column": "log_addr",
           "filter_op": "contains",
-          "filter_arg": ["0x3Fa45962C5090b06394331467808751E77E9Ef57"]
+          "filter_arg": ["0xFF42991c567c291eD9c2Ccc663cDC9fe36Eef8e0"]
```

Comment:
what's this address? the deployed Bodies address? Shouldn't we be pulling this automatically so we don't need to update it each time?

Reply:
this file re-generated by `bun run shovel`. could gitignore it but I think it's useful for debugging.

Reply:
removing it and gitignoring so as to not expose rpc urls